### ISSUE TITLE
[Enhancement]: 

### ISSUE THREAD

**Issue by lathspell** [hashicorp/terraform-provider-aws#29164]
### Description

Please offer a way to disable SecurityHub findings for certain resources.
It should be possible using something similar to

```
     aws securityhub batch-update-findings \
        --finding-identifiers '[ \
            { \
              "Id": "arn:aws:securityhub:eu-central-1:12345:subscription/aws-foundational-security-best-practices/v/1.0.0/S3.11/finding/12345678-6af9-4bf2-83e7-5e00b7e456a3", \
              "ProductArn": "arn:aws:securityhub:us-west-1::product/aws/securityhub" \
            } \
          ]' \
      --note '{"Text": "Known issue that is not a risk.", "UpdatedBy": "terraform"}' \
      --workflow '{"Status": "RESOLVED"}'
```

### Potential Terraform Configuration

```terraform
resource "aws_securityhub_findings" "foo" {
  standards_control_arn = "arn:aws:securityhub:eu-central-1:385122011390:subscription/aws-foundational-security-best-practices/v/1.0.0/S3.9/123123"
  workflow_status       = "SUPPRESSED"
  reason                = "We do not need that!"
}
```

### Would you like to implement a fix?

None

**Comment by nitrocode** [comment]
@lathspell @justinretzolk could you folks retitle this to be `[Enhancement]: aws_securityhub_batch_update_findings for suppression`

---

Here's the full awscli doc on `aws securityhub batch-update-findings`

https://awscli.amazonaws.com/v2/documentation/api/latest/reference/securityhub/batch-update-findings.html

**Comment by lathspell** [comment]
Renamed. Thanks for providing the links.

**Comment by nitrocode** [comment]
@lathspell I've been using this terraform module and it's been handy at codifying suppressions. This is a stop-gap until there is a native terraform resource.

https://registry.terraform.io/modules/infralicious/securityhub-batchupdatefindings/aws/latest

TLDR: It uses the awscli with a yaml file of suppressions with `--note` to suppress each finding with a note.
